blocks/rich-text/index.js

```diff
@@ -412,14 +395,25 @@ export class RichText extends Component {
 	 *
 	 * @param {boolean} checkIfDirty Check whether the editor is dirty before calling onChange.
 	 */
-	onChange( checkIfDirty = true ) {
```

Comment:
Since we're removing the argument, we should also remove the JSDoc tag.

Reply:
Oh. Surprised there's no listing error for this one.